src/vivarium_gates_shigella_vaccine/data/standardize.py

```diff
@@ -45,120 +45,6 @@ def normalize_year(data: pd.DataFrame) -> pd.DataFrame:
     return data
 
 
-def normalize_for_simulation(df):
```

Comment:
Turns out I don't need any of this stuff.

Reply:
at this point, isn't a bunch of the previous code using it ?